ner_v1/api.py

```diff
@@ -94,6 +93,7 @@ def parse_post_request(request):
         PARAMETER_BOT_MESSAGE: request_data.get('bot_message'),
         PARAMETER_TIMEZONE: request_data.get('timezone'),
         PARAMETER_REGEX: request_data.get('regex'),
+        PARAMETER_ASR: request_data.get('is_asr'),
```

Comment:
Can we also assign a default value of this to False like in Line_no:63

Reply:
Hey Tanaya, I was suggesting this for consistency across the file, In line number 63 of the same file, we have  `PARAMETER_ASR: request.GET.get('is_asr', False),`.

Reply:
Discussed over call, the other dictionary where Line no: 63 had a default value assigned had some other request params with default value being assigned, so it was added there and not here. Marking resolved.